README.md

```diff
@@ -61,6 +61,47 @@ or by using the automatic discovery.
 
 This will also work for all other devices as long as the device has been bound with the Mi Home mobile application, and is therefore preferable way to attain the token.
 
+### miio-extract-tokens
+
+The package installs also a tool to simplify the extraction of tokens.
```

Comment:
The package provides a command line tool to extract tokens from android backups and SQlite databases. Please follow the above-mentioned procedure to retrieve the tar file (Android) or a SQlite database (Android & Apple). Encrypted tokens as recently introduced on iOS devices will be automatically decrypted.

Reply:
Sounds much better, I just uppercased Android in the first sentence. Thanks!